pandas/tests/io/pytables/test_round_trip.py

```diff
@@ -349,15 +349,17 @@ def test_index_types(setup_path):
         _check_roundtrip(ser, func, path=setup_path)
 
 
-def test_timeseries_preepoch(setup_path):
+def test_timeseries_preepoch(setup_path, request):
 
     dr = bdate_range("1/1/1940", "1/1/1960")
     ts = Series(np.random.randn(len(dr)), index=dr)
     try:
         _check_roundtrip(ts, tm.assert_series_equal, path=setup_path)
     except OverflowError:
         if is_platform_windows():
-            pytest.xfail("known failure on some windows platforms")
+            request.node.add_marker(
+                pytest.mark.xfail("known failure on some windows platforms")
+            )
         else:
```

Comment:
Nit: I think this `else` should be removed now too so the xfail will be recognized